.github/workflows/topos:integration-tests.yml

```diff
@@ -31,7 +31,8 @@ jobs:
     runs-on: ubuntu-latest-16-core
     environment: devnet-1
     env:
-      INFRA_REF: ${{ inputs.local-erc20-messaging-infra-ref || needs.infra-metadata.outputs.latest-release }}
+      # INFRA_REF: ${{ inputs.local-erc20-messaging-infra-ref || needs.infra-metadata.outputs.latest-release }}
+      INFRA_REF: "chore/debug-ci-1"
```

Comment:
This needs to be removed

Reply:
@Freyskeyd Is this here temporarily to test it with a pending PR? Is there an alternative to this?

Reply:
@Freyskeyd This is the alternative
![image](https://github.com/topos-protocol/e2e-tests/assets/12594650/72be5a2a-4a1a-4c6a-a74b-1232c5544adf)